code/datums/components/foldable.dm

```diff
@@ -2,19 +2,23 @@
 	var/obj/item/objBriefcase/the_briefcase
 	var/original_name
 	var/original_desc
+	var/briefcase_path
 	var/change_name = 1
 
 /datum/component/foldable/keep_name
 	change_name = 0
 
-/datum/component/foldable/Initialize()
+/datum/component/foldable/Initialize(var/briefcase_path = /obj/item/objBriefcase)
 	if(!istype(parent, /atom/movable))
 		return COMPONENT_INCOMPATIBLE
+	if(!ispath(briefcase_path,/obj/item.objBriefcase))
```

Comment:
```suggestion
	if(!ispath(briefcase_path, /obj/item/objBriefcase))
```

Reply:
Re: this one, could you like... actually make the change I asked for? 👀

Reply:
oh, ya, sorry thought you meant the space